lib/cryptoexchange/exchanges/bitconnect/services/market.rb

```diff
@@ -0,0 +1,52 @@
+module Cryptoexchange::Exchanges
+  module Bitconnect
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Bitconnect::Market::API_URL}/info"
+        end
+
+        def adapt_all(output)
+          markets = output['markets']
+          markets.map do |value|
+            base, target = value['marketname'].split('_')
```

Comment:
Should be `target, base` since we want target to be BTC and base to be BCC

Reply:
Corrected to `target`, `base`